README.md

```diff
@@ -1,3 +1,10 @@
+[![Build Status](https://img.shields.io/jenkins/build?jobUrl=http%3A%2F%2Fjenkins.iudx.io%3A8080%2Fjob%2Fiudx%2520aaa-server%2520%28master%29%2520pipeline%2F)](http://jenkins.iudx.io:8080/job/iudx%20aaa-server%20(master)%20pipeline/lastBuild/)
+[![Jenkins Coverage](https://img.shields.io/jenkins/coverage/jacoco?jobUrl=http%3A%2F%2Fjenkins.iudx.io%3A8080%2Fjob%2Fiudx%2520aaa-server%2520%28master%29%2520pipeline%2F)](http://jenkins.iudx.io:8080/job/iudx%20aaa-server%20(master)%20pipeline/lastBuild/jacoco/)
+[![Unit Tests](https://img.shields.io/jenkins/tests?jobUrl=http%3A%2F%2Fjenkins.iudx.io%3A8080%2Fjob%2Fiudx%2520aaa-server%2520%28master%29%2520pipeline%2F&label=unit%20tests)](http://jenkins.iudx.io:8080/job/iudx%20aaa-server%20(master)%20pipeline/lastBuild/testReport/)
+[![Security Tests](https://img.shields.io/jenkins/build?jobUrl=http%3A%2F%2Fjenkins.iudx.io%3A8080%2Fjob%2Fiudx%2520aaa-server%2520%28master%29%2520pipeline%2F&label=security%20tests)](http://jenkins.iudx.io:8080/job/iudx%20aaa-server%20(master)%20pipeline/lastBuild/zap/)
+[![Integration Tests](https://img.shields.io/jenkins/build?jobUrl=http%3A%2F%2Fjenkins.iudx.io%3A8080%2Fjob%2Fiudx%2520aaa-server%2520%28master%29%2520pipeline%2F&label=integration%20tests)](http://jenkins.iudx.io:8080/job/iudx%20aaa-server%20(master)%20pipeline/Integration_20Test_20Report/)
```

Comment:
For the integration tests, it says that it is passing (when you view the file) though 20 are failing, is that OK? Or is it possible to get the same sort of _x_ passed / _y_ failed like the unit tests have

Reply:
Since these 20 tests are known to fail, I had to catch the error in that block so that the pipeline doesn't fail ( because we cannot evaluate the html report and set a failure threshold higher than known failures) .If we want to reflect the correct status on the badges, we can remove the try/catch but that would also fail the pipeline. 
And since the git badge reflects the status of the pipeline and not the actual integration test stage, it shows that they are passing currently. Unfortunately there are no plugins available to get a badge for newman reports that's why we had to go with this approach.

Reply:
OK, thanks Karun. I will work on updating the collection to try and fix the failing tests. I'll let you know when that's done, and then the try/catch can be removed